I added a config like this:
```
    def configure_vars(self):
        args = ['LIB_DIR=%s' % self.spec['mariadb-client'].prefix.lib.mariadb,
                'INCLUDE_DIR=%s' % self.spec['mariadb-client'].prefix.include.mariadb]                                                                                                                                                           
        return args
```

But it still cannot work, any sugesttion for that?
```
==> r-rmariadb: Executing phase: 'install'
==> [2021-02-02-14:23:28.673051] '/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/bin/R' 'CMD' 'INSTALL' '--configure-vars=LIB_DIR=/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/lib/mariadb INCLUDE_DIR=/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb' '--library=/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library' '/home/stage/root/spack-stage-r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/spack-src'
* installing *source* package 'RMariaDB' ...
** package 'RMariaDB' successfully unpacked and MD5 sums checked
** using staged installation
Using mariadb_config
WARNING: Connector versions prior to 2.3.4/3.0.3 have a known problem with retrieving string and blob data. See https://github.com/r-dbi/RMariaDB/issues/53 for details.
Found INCLUDE_DIR and/or LIB_DIR!
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:172: warning: overriding recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:172: warning: ignoring old recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:174: warning: overriding recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:174: warning: ignoring old recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:177: warning: overriding recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:177: warning: ignoring old recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:179: warning: overriding recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:179: warning: ignoring old recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:181: warning: overriding recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:181: warning: ignoring old recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:183: warning: overriding recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:183: warning: ignoring old recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:186: warning: overriding recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:186: warning: ignoring old recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:189: warning: overriding recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:189: warning: ignoring old recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:191: warning: overriding recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:191: warning: ignoring old recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:193: warning: overriding recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:193: warning: ignoring old recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:195: warning: overriding recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:195: warning: ignoring old recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:198: warning: overriding recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:198: warning: ignoring old recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:200: warning: overriding recipe for target '.f90.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:200: warning: ignoring old recipe for target '.f90.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:172: warning: overriding recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:172: warning: ignoring old recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:174: warning: overriding recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:174: warning: ignoring old recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:177: warning: overriding recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:177: warning: ignoring old recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:179: warning: overriding recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:179: warning: ignoring old recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:181: warning: overriding recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:181: warning: ignoring old recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:183: warning: overriding recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:183: warning: ignoring old recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:186: warning: overriding recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:186: warning: ignoring old recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:189: warning: overriding recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:189: warning: ignoring old recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:191: warning: overriding recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:191: warning: ignoring old recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:193: warning: overriding recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:193: warning: ignoring old recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:195: warning: overriding recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:195: warning: ignoring old recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:198: warning: overriding recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:198: warning: ignoring old recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:200: warning: overriding recipe for target '.f90.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:200: warning: ignoring old recipe for target '.f90.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:172: warning: overriding recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:172: warning: ignoring old recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:174: warning: overriding recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:174: warning: ignoring old recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:177: warning: overriding recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:177: warning: ignoring old recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:179: warning: overriding recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:179: warning: ignoring old recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:181: warning: overriding recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:181: warning: ignoring old recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:183: warning: overriding recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:183: warning: ignoring old recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:186: warning: overriding recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:186: warning: ignoring old recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:189: warning: overriding recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:189: warning: ignoring old recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:191: warning: overriding recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:191: warning: ignoring old recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:193: warning: overriding recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:193: warning: ignoring old recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:195: warning: overriding recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:195: warning: ignoring old recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:198: warning: overriding recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:198: warning: ignoring old recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:200: warning: overriding recipe for target '.f90.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:200: warning: ignoring old recipe for target '.f90.o'
Using PKG_CFLAGS=-I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb 
Using PKG_LIBS=-L/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/lib/mariadb -lmysqlclient
** libs
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:172: warning: overriding recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:172: warning: ignoring old recipe for target '.c.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:174: warning: overriding recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:174: warning: ignoring old recipe for target '.c.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:177: warning: overriding recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:177: warning: ignoring old recipe for target '.m.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:179: warning: overriding recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:179: warning: ignoring old recipe for target '.cc.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:181: warning: overriding recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:181: warning: ignoring old recipe for target '.cpp.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:183: warning: overriding recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:183: warning: ignoring old recipe for target '.cc.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:186: warning: overriding recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:186: warning: ignoring old recipe for target '.cpp.d'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:189: warning: overriding recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:189: warning: ignoring old recipe for target '.m.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:191: warning: overriding recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:191: warning: ignoring old recipe for target '.mm.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:193: warning: overriding recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:193: warning: ignoring old recipe for target '.M.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:195: warning: overriding recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:195: warning: ignoring old recipe for target '.f.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:198: warning: overriding recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:198: warning: ignoring old recipe for target '.f95.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf.spack:200: warning: overriding recipe for target '.f90.o'
/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/etc/Makeconf:200: warning: ignoring old recipe for target '.f90.o'
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c DbConnection.cpp -o DbConnection.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c DbResult.cpp -o DbResult.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaBinding.cpp -o MariaBinding.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaResult.cpp -o MariaResult.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaResultImpl.cpp -o MariaResultImpl.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaResultPrep.cpp -o MariaResultPrep.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaResultSimple.cpp -o MariaResultSimple.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaRow.cpp -o MariaRow.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaTypes.cpp -o MariaTypes.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c MariaUtils.cpp -o MariaUtils.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c connection.cpp -o connection.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c driver.cpp -o driver.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -I"/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/include" -DNDEBUG -I/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb  -DRCPP_DEFAULT_INCLUDE_CALL=false -DRCPP_USING_UTF8_ERROR_STRING -DBOOST_NO_AUTO_PTR -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-bh-1.72.0-3-hrm6m54msmyr4m4dnpsu6c7mz667qogx/rlib/R/library/BH/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-plogr-0.2.0-ira3656toukb7cqqjym23o7rn7wgqlu7/rlib/R/library/plogr/include' -I'/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rcpp-1.0.4.6-vfvmowoxiccg7x6kyr6fq35ykwjqgd27/rlib/R/library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c result.cpp -o result.o
/home/spack/lib/spack/env/gcc/g++ -std=gnu++11 -shared -L/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/lib -L/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/lib -Wl,-rpath,/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/lib -o RMariaDB.so DbConnection.o DbResult.o MariaBinding.o MariaResult.o MariaResultImpl.o MariaResultPrep.o MariaResultSimple.o MariaRow.o MariaTypes.o MariaUtils.o RcppExports.o connection.o driver.o result.o -L/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/lib/mariadb -lmysqlclient -L/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/rlib/R/lib -lR
installing to /home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library/00LOCK-spack-src/00new/RMariaDB/libs
** R
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for 'RMariaDB' in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library/00LOCK-spack-src/00new/RMariaDB/libs/RMariaDB.so':
  libmariadb.so.3: cannot open shared object file: No such file or directory
Error: loading failed
Execution halted
ERROR: loading failed
* removing '/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library/RMariaDB'
==> Error: ProcessError: Command exited with status 1:
    '/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-4.0.3-rrhlzbbvznxhirnfag66fwfach6a77ma/bin/R' 'CMD' 'INSTALL' '--configure-vars=LIB_DIR=/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/lib/mariadb INCLUDE_DIR=/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/mariadb-c-client-3.1.9-p634wzoiiwxvtgkmzoejdbkjcpav5w5g/include/mariadb' '--library=/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library' '/home/stage/root/spack-stage-r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/spack-src'

3 errors found in build log:
     132    ** R
     133    ** byte-compile and prepare package for lazy loading
     134    ** help
     135    *** installing help indices
     136    ** building package indices
     137    ** testing if installed package can be loaded from temporary location
  >> 138    Error: package or namespace load failed for 'RMariaDB' in dyn.load(file, DLLpath = DLLpath, ...):
     139     unable to load shared object '/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library/00LOCK-spack-src/00new/RMariaDB/libs/RMariaDB.so':
  >> 140      libmariadb.so.3: cannot open shared object file: No such file or directory
  >> 141    Error: loading failed
     142    Execution halted
     143    ERROR: loading failed
     144    * removing '/home/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/rlib/R/library/RMariaDB'

See build log for details:
  /home/stage/root/spack-stage-r-rmariadb-1.0.8-dqxjmgknej6jkeu3vkglpkcdsc5dkgli/spack-build-out.txt
```

--- a/var/spack/repos/builtin/packages/r-rmariadb/configure_add_rpath.patch
+++ b/var/spack/repos/builtin/packages/r-rmariadb/configure_add_rpath.patch
@@ -1,0 +1,13 @@
+diff --git a/configure b/configure
+index 6c0f43c..bd7f627 100755
+--- a/configure
++++ b/configure
+@@ -29,7 +29,7 @@ fi
+ if [ "$INCLUDE_DIR" ] || [ "$LIB_DIR" ]; then
+   echo "Found INCLUDE_DIR and/or LIB_DIR!"
+   PKG_CFLAGS="-I$INCLUDE_DIR $PKG_CFLAGS"
+-  PKG_LIBS="-L$LIB_DIR $PKG_LIBS"
++  PKG_LIBS="-L$LIB_DIR -Wl,-rpath,$LIB_DIR $PKG_LIBS"
+ elif [ "$PKGCONFIG_CFLAGS" ] || [ "$PKGCONFIG_LIBS" ]; then
+   echo "Found mysql_config/mariadb_config cflags and libs!"
+   PKG_CFLAGS=${PKGCONFIG_CFLAGS}

--- a/var/spack/repos/builtin/packages/r-rmariadb/package.py
+++ b/var/spack/repos/builtin/packages/r-rmariadb/package.py
@@ -31,6 +31,8 @@ class RRmariadb(RPackage):
     patch('configure_add_rpath.patch')
 
     def configure_vars(self):
-        args = ['LIB_DIR={0}'.format(self.spec['mariadb-client'].prefix.lib.mariadb),
-                'INCLUDE_DIR={0}'.format(self.spec['mariadb-client'].prefix.include.mariadb)]
+        lib_dir = self.spec['mariadb-client'].prefix.lib.mariadb
+        inc_dir = self.spec['mariadb-client'].prefix.include.mariadb
+        args = ['LIB_DIR={0}'.format(lib_dir),
+                'INCLUDE_DIR={0}'.format(inc_dir)]
         return args

--- a/var/spack/repos/builtin/packages/r-rmariadb/package.py
+++ b/var/spack/repos/builtin/packages/r-rmariadb/package.py
@@ -28,6 +28,9 @@ class RRmariadb(RPackage):
     # non-R dependencies
     depends_on('mariadb-client')
 
-    def setup_build_environment(self, env):
-        env.prepend_path('LD_LIBRARY_PATH',
-                         self.spec['mariadb-client'].prefix.lib.mariadb)
+    patch('configure_add_rpath.patch')
+
+    def configure_vars(self):
+        args = ['LIB_DIR={0}'.format(self.spec['mariadb-client'].prefix.lib.mariadb),
+                'INCLUDE_DIR={0}'.format(self.spec['mariadb-client'].prefix.include.mariadb)]
+        return args

